### DISCUSSION TITLE
Server side copy with snowball extraction

### DISCUSSION THREAD

**Discussion by comassky** [minio/minio#17818] [Q&A] [answered]
Hi,

We want to make a server-side copy (CopyObjectRequest) of a tar.gz archive, and we would like to  the destination archive to be decompressed (via snowball).

The snowball headers (snowball-auto-extract) don't seem to be working; according to our code analysis, the Metadata map is set to null in processing.

Is there a way to copy/move an archive file and have it decompressed in the destination bucket without having to retrieve the file + resend it via the client?

**Comment by harshavardhana** [comment, marked as answer]
Unfortunately no we do not support it yet. Feel free to send a PR
